research/berry/berrylib/test_berry.py

```diff
@@ -200,8 +202,8 @@ def test_fast_inla(method, N=10, iterations=1):
 
     # Do our comparisons in probability space.
     np.testing.assert_allclose(
-        logistic(theta_max[0, 12]),
```

Comment:
This logistic is correct I thought?

Reply:
Fair enough, though do please remove `# Do our comparisons in probability space.`